test/integration/patients/worklist/worklist.js

```diff
@@ -3761,7 +3768,7 @@ context('worklist page', function() {
         const flowInclude = {
           id: '1',
           type: 'flows',
-          attributes: _.extend(_.sample(this.fxFlows), {
+          attributes: _.extend(_.sample(fxFlows), {
```

Comment:
even this is mutating..  I suppose we need to figure out how imported fixtures work in cypress an go from there.